zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchExporterIT.java

```diff
@@ -58,6 +70,8 @@ final class ElasticsearchExporterIT {
   private final RecordIndexRouter indexRouter = new RecordIndexRouter(config.index);
 
   private TestClient testClient;
+  private ExporterTestContext exporterTestContext;
+  private int exportedRecordsCounter = 0;
```

Comment:
❌  I would prefer to clear elastic after/before each test instead of handling test side effects via this counter. Then you can be sure you always have clean state and not worry about what other tests do.
```
  @BeforeEach
  void cleanup() {
    testClient.deleteIndices();
  }
```

with extending TestClient with https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-index.html which will also require you to set the env var`action.destructive_requires_name=false` on the test container so you can do wildcard deletes.

Reply:
thanks a lot! that's a great hint!